### ISSUE TITLE
[Bug report] Warnings/Message boxes don't render correctly on the latest version of Firefox

### ISSUE THREAD

**Issue by xplshn** [zshall/shell-css#1]
![2024-02-04-224653_830x540_scrot](https://github.com/zshall/shell-css/assets/114888778/f5899ca6-788c-4420-9f27-3214ca38a958)
This is the only browser which I have installed in my PC. So I can't really compare to others, however, I don't think it is supposed to look like that:
Otherwise, really cool project! I will probably make a GTK themese based off of this

**Comment by zshall** [comment]
Thanks for your report! It appears that this is related to breakpoints in `@media` queries; in Firefox 122 on Windows it looks like this for me at 1920w:

![image](https://github.com/zshall/shell-css/assets/175742/e8eff982-3a23-473f-85b7-988220362a57)

I can reproduce the bug you described in WebKit, at a width of 1200-1400 pixels, so it's very likely related to this:

![image](https://github.com/zshall/shell-css/assets/175742/bcd7e52e-6ae5-45f6-8567-c175404ca780)

Let me take a look and see what I can find; it's probably going to regard padding on the icon on the left.

As far as GTK theming that sounds great! I envision a future version of this project where you can declare `<div class="theme-default shell-gtk">` or `shell-3.1` or `shell-os9`, etc. and we would adapt to it. We'll probably want to reorganize the folder structure to reflect this. I'll write up an issue about getting this done.

**Comment by zshall** [comment]
I've taken a look and implemented some fixes. Added a `.btn-group` class for dealing with multiple buttons next to each other which spaces them out using `gap`. I also updated the documentation to add `.p-0` classes to the left hand side of alerts so that they work in constrained screen sizes; this also makes them look better positioned. Here's what it looks like on Firefox:

![image](https://github.com/zshall/shell-css/assets/175742/436bb5d6-dd10-498e-9b41-3681eb904392)
